parser-typechecker/src/Unison/Codebase/TranscriptParser.hs

```diff
@@ -68,7 +69,7 @@ instance Show Stanza where
   show s = case s of
     Ucm _ _ cmds -> unlines [
       "```ucm",
-      show cmds,
+      foldl (\x y -> x ++ show y) "" cmds,
```

Comment:
This means that when showing a complete failing stanza, there are no angle braces around each command